src/OpenTelemetry.Instrumentation.AWS/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.1.1-beta.1
```

Comment:
This package has not had a stable `1.1.0` release yet.

```suggestion
## 1.1.0-beta.2
```

Reply:
Fixed